packages/mdc-grid-list/foundation.ts

```diff
@@ -44,8 +44,10 @@ class MDCGridListFoundation extends MDCFoundation<MDCGridListAdapter> {
   private readonly resizeHandler_: EventListener;
   private resizeFrame_ = 0;
 
-  constructor(adapter: MDCGridListAdapter) {
-    super(Object.assign(MDCGridListFoundation.defaultAdapter, adapter));
+  /* istanbul ignore next: optional argument is not a branch statement */
```

Comment:
Is `ignore next` still necessary without a default assignment? I don't see it on the other constructors using an optional argument.

Reply:
I'm honestly not sure why it's needed in some places but not others.

I've noticed that it's _usually_ needed when a base foundation is subclassed by another foundation that doesn't have its own constructor (e.g., tab-scroller), but this case I have no idea.

My only guess is that it's a quirk in Istanbul.